index.html

```diff
@@ -10,10 +10,98 @@
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
 <body>
-
+    
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    
+    <main>
+        <header class="header">
+            <img class="fav_ico" src="img/karma-logo.svg"/>
+            
+            <div>          
+                <nav class="nav_ul">
+                    <a href="#">Meet Karma</a>
+                    <a href="#">How it Works</a>
+                    <a href="#">Store</a>
+                    <a href="#">Blog</a>
+                    <a href="#">Help</a>
+                    <a href="#">Login</a>
+                </nav>
+            </div>
+            
+        </header>
+    </main>    
+    <main class="main_back_img">
+        <div class="ad_back_img">   
+            <h3>introducing Karma</h3>
+            <h4>Bring WiFi with you, everywhere you go</h4>
+            <button class="learn_button">Learn More</button>
+        </div>
+             
+    </main>
+    <main class="Every_needs">
+        <h1>Everyone needs a little Karma</h1>
+    </main>
+    <main class="three_imgs">
+        <div class="img_device img_1">
+            <img  src="img\icon-devices.svg" alt="icon devices">
+            <p>internet for all devices</p>
+        </div>
+        <div class="img_coffee img_1">
+            <img src="img/icon-coffee.svg" alt="icon divices">
+            <p>iBoost your productivity</p>
+        </div>
+        <div class="img_refill img_1">
+            <img src="img/icon-refill.svg" alt="icon devices">
+            <p>Pay as you Go</p>
+        </div>
+
+        
+    </main>
+    <div class="contener">
+        <img class="contener_img" src="level-2/homepage-feature.png" alt="img picture">
+        <div class="block_text">
+            <h1 class="quote"><span class="quote_mark">&ldquo;</span>Wherever I am,I just don't <br>
+                worry about my connection<br> anymore!<span class="quote_mark">&rdquo;</span></h1>
+            <a href="css/danti.html"><button class="get_garma_1">Get Karma totay</button></a>    
+        </div>
+    </div>
+    <hr>
+    
+    
+    <!-- <header class="header">
+        
+            <img class="img1" src="favicon.ico"/>            
+            <a href="#">Meet Karma</a>
+            <a href="#">How it Works</a>
+            <a href="#">Store</a>
+            <a href="#">Blog</a>
+            <a href="#">Help</a>
+            <a href="#">Login</a>
+       
+        <div>
+            <img src="first-background.jpg"/>
+        </div> -->
+           
+    </header>
```

Comment:
It looks like you missed this closing tag when commenting out the block above - it can be removed along with the commented out code.

As an extra note, this extra closing tag affects the validity of the HTML page. Browsers are usually quite forgiving, and this probably won't cause any visual or functional issues with your page, but it will stop Prettier from formatting your code - Prettier can only format your code if it is valid. If you remove this closing tag and then run the "format document" command in VSCode with this file open then you'll see several changes occur, like extra spaces being removed, indentation changed, etc. You can run the `format document` command with `ctrl` + `shift` + `p` (or `cmd` + `shift` + `p` if you're using a Mac), then type `format document` and hit enter. If you've set up VSCode as described [here](https://docs.codeyourfuture.io/Leaders/joining-the-course/getting-setup#5.3-enable-formatting-on-save) then this should happen automatically when you save your code.